x-pack/plugins/canvas/public/components/app/index.tsx

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FC, useRef, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { History } from 'history';
+// @ts-expect-error
+import createHashStateHistory from 'history-extra/dist/createHashStateHistory';
```

Comment:
Um... we should replace this soon, too.

https://www.npmjs.com/package/history-extra